Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM cfssl/cfssl:1.3.2 as cfssl
+FROM elixir:1.6
+
+ENV \
+  LANG=C.UTF-8 \
+  LC_ALL=en_US.UTF-8 \
+  PATH="/app:${PATH}"
+
+ADD . /app
+WORKDIR /app
+
+COPY --from=cfssl /go/bin/cfssl cfssl
+
+RUN set -xe \
```

Comment:
You don't need the `set -e` if the commands are strung together with `&&`. I wonder if the commands were originally connected with `;`'s. That would make more sense. Not a big deal, though.